controllers/nginx/Makefile

```diff
@@ -3,7 +3,7 @@ all: push
 BUILDTAGS=
 
 # 0.0 shouldn't clobber any release builds
```

Comment:
that's saying if you want to push some test build to gcr to eg: test your write access, use 0.0. I suppose I can actually say that.